compose/runtime/runtime/build.gradle

```diff
@@ -175,6 +176,8 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
             desktopMain.dependsOn(jbMain)
             jsNativeMain.dependsOn(jbMain)
 
+            linuxArm64Main.dependsOn(nativeMain)
+            linuxArn64Test.dependsOn(nativeTest)
```

Comment:
```suggestion
            linuxArm64Test.dependsOn(nativeTest)
```

Reply:
Oof thanks!

Reply:
Had to remove you as a co-author since it used your opaque GitHub email and there's no Google CLA for that.